src/shared/components/input/Input.scss

```diff
@@ -1,23 +1,56 @@
 @import 'src/styles/common';
 
 .input {
-  display: inline-block;
-  width: 100%;
+  &::placeholder {
+    color: $dark-grey;
+    font-weight: $light;
+  }
 
   &:focus,
   &:active {
     outline: none;
   }
+}
 
-  &::placeholder {
-    color: $black;
-    font-weight: $light;
+.rightCorner {
+  display: flex;
+  align-items: center;
+  flex: 0 0 auto;
+}
+
+.inputWrapper {
+  background-color: var(--input-background-colour, $white);
+  display: inline-flex;
+  padding: 0 20px;
+  width: var(--input-width, 100%);
+
+  input {
+    background-color: transparent;
+    flex-grow: 1;
+    border: none;
+    box-shadow: none;
   }
 }
 
-.shadedInput {
-  box-shadow: $form-field-shadow;
+.shadedInputWrapper {
   border: none;
-  padding: 0 15px;
-  height: 30px;
+  box-shadow: $form-field-shadow;
+  height: var(--shaded-input-height, 40px);
```

Comment:
Out of curiosity — which shaded input is going to be more common: 30px high or 40px high (depending on this, we will pick our default). What will Andrea say if we ask her?

Reply:
![image](https://user-images.githubusercontent.com/6834224/231502382-f81acbc4-5506-4c19-b405-580709a3522d.png)

Let's do 30 as a default.

Reply:
✅